spec/rubocop/cop/style/redundant_parentheses_spec.rb

```diff
@@ -143,4 +143,16 @@
     inspect_source(cop, 'if x; y else (1)end')
     expect(cop.offenses).to be_empty
   end
+
+  context 'when a hash literal is the first argument in a method call' do
+    it 'accepts parentheses if the argument list is not parenthesized ' do
+      inspect_source(cop, 'x ({ y: 1 }), z')
```

Comment:
It is intentional, this matches the format where this cop used to give an offence, as describe [here](https://github.com/bbatsov/rubocop/issues/2729#issue-129258790).